### ISSUE TITLE
Default repository

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/windows-package-manager#333]
```
What steps will reproduce the problem?
1. Delete default repository list
2. (Restart Npackd)

What is the expected output? What do you see instead?
After a restart, the default repository URLs are no longer present in the 
settings, but Npackd will still show and install software from these. I'd like 
to use Npackd for custom repositories only.

What version of the product are you using? On what operating system?
Npackd 1.18.7 64-bit on Windows 7 x64 with all updates installed  

Please provide any additional information below.

```

Original issue reported on code.google.com by `_...@whyeye.org` on 18 Feb 2014 at 8:31

**Comment by GoogleCodeExporter** [comment]
```
I cannot reproduce it. If I try to remove all repositories, I get an error 
message, but if I add another one everything works as expected.
```

Original comment by `tim.lebe...@gmail.com` on 19 Feb 2014 at 9:22

Attachments:
* [error.png](https://storage.googleapis.com/google-code-attachments/windows-package-manager/issue-333/comment-1/error.png)

**Comment by GoogleCodeExporter** [comment]
```
It works as desired in the two virtual machines I tested it (XP x86, Win7 x64), 
so I guess the mistake is on my end. Still kinda weird.
```

Original comment by `_...@whyeye.org` on 22 Feb 2014 at 8:26

**Comment by GoogleCodeExporter** [comment]
```
this was fixed in 1.18. Have you used 1.17 before?
```

Original comment by `tim.lebe...@gmail.com` on 23 Feb 2014 at 12:52

**Comment by GoogleCodeExporter** [comment]
```
this was fixed in 1.18. Have you used 1.17 before?
```

Original comment by `tim.lebe...@gmail.com` on 23 Feb 2014 at 12:52

**Comment by GoogleCodeExporter** [comment]
Original comment by `tim.lebe...@gmail.com` on 26 Feb 2014 at 7:50
* Changed state: **Invalid**
